lib/src/blocs/movies_bloc.dart

```diff
@@ -19,9 +19,26 @@ class MoviesBloc extends IMoviesBloc {
   @override
   void fetchMoviesApi() async {
```

Comment:
You can change this function with a more descriptive name, remember that don't reference to Api. You can use the same approach of _movieRepository fetch call (fetchAllMovies)

Reply:
Thank you for remembering me that. More descriptive name added.